blue_modules/notifications.js

```
@@ -9,506 +10,528 @@
+const configureNotifications = async onProcessNotifications => {
+  return new Promise(resolve => {
+    requestNotifications(['alert', 'sound', 'badge']).then(({ status, _ }) => {
+      if (status === 'granted') {
+        PushNotification.configure({
+          onRegister: async token => {
+            cachedIsNotificationsEnabled = true;
+            resolve(true);
+          },
+          onNotification: async notification => {
+            const payload = Object.assign({}, notification, notification.data);
+            if (notification.data && notification.data.data) Object.assign(payload, notification.data.data);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)